requirements.txt

```diff
@@ -14,6 +14,7 @@ django-capture-tag
 django-cas-ng
 django-countries
 django-crispy-forms
+django-extensions
```

Comment:
Do we want it in `requirements.txt` or rather `requirements-dev.txt`?

Reply:
Makes sense, forgot about this one.